float32 first_coordinate
float32 second_coordinate
int32 to_polar
---
float32 first_converted_coordinate
float32 second_converted_coordinate